src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -525,20 +559,27 @@ function updateConstSumQnInstructions(qnNum) {
         points *= numOptions;
     }
 
+    // Specifies the type of distribution constraint selected by question creator
```

Comment:
`Specifies if the constraint for uneven distribution of points is selected` or something simpler